mailinglogger/mailinglogger.py

```diff
@@ -109,6 +111,9 @@ def emit(self, record):
             email['Message-ID'] = make_msgid('MailingLogger')
             smtp = smtplib.SMTP(self.mailhost, self.mailport)
             if self.username and self.password:
+                if self.secure is not None:
+                    smtp.starttls(*self.secure)
+                    smtp.ehlo()
```

Comment:
For the reasons I covered in https://github.com/simplistix/mailinglogger/pull/23, I'm not a fan of this.

https://docs.python.org/3/library/smtplib.html#smtplib.SMTP.starttls suggests usage of `starttls()` is one of:

- `starttls()`
- `starttls(keyfile, certfile)`
- `starttls(context=SSLContext(...)`

Which of these are you actually planning to use?

Reply:
At the moment, all settings would be supported, which makes the setup and API more complicated as required. I could simplify this and rename `secure` to `use_tls` in the MailingLogger constructor, and then call `smtp.starttls()` without parameters.

In my projects, I am using only the standard `starttls()` without parameters.